src/structured_data.rs

```diff
@@ -166,10 +166,8 @@ impl StructuredData {
         let data = try!(self.data_to_sign());
         // Count valid previous_owner_signatures and refuse if quantity is not enough
 
-        let check_all_keys = |&sig| {
-            owner_keys.iter()
-                .any(|pub_key| sign::verify_detached(&sig, &data, pub_key))
-        };
+        let check_all_keys =
+            |&sig| owner_keys.iter().any(|pub_key| verify_detached(&sig, &data, pub_key));
```

Comment:
Yes, but isn't that expected behaviour? Wouldn't it rather be surprising if four valid signatures don't suffice because the fifth one is `NO_OWNER_PUB_KEY`?
I assumed the idea was that to permanently lock a chunk, the user would make this one the only owner.